### DISCUSSION TITLE
Game protection that don't work

### DISCUSSION THREAD

**Discussion by Altheos** [86Box/86Box#1754] [General]
Hi,
as suggested, i'm starting this discussion to list protected games that can't be launched : 

- Microprose protection 2 (F-15 Strike Eagle II / Red storm rising ..)
- Microprose protection 1 (Gunship)
- Titus Megaprot (Highway Patrol II)
- Softguard SUPERLoK 2.x/3.x (Arkanoid II, Sky Shark ..)
- Waydisk minder (Football Manager, Frank Bruno's boxing ..)
- Formaster Copylock (Block Buster)
- Electronic Arts IBM Interlock (Super Boulder Dash)
- Unknown : King of Chicago
- Booter : Bruce Lee

**Comment by Olddantrucker** [comment]
Midtown Madness II (Microsoft)
Need For Speed High Stakes (EA games)
